src/main/java/org/hid4java/jna/DarwinHidApiLibrary.java

```diff
@@ -0,0 +1,23 @@
+package org.hid4java.jna;
+
+import com.sun.jna.Native;
+import com.sun.jna.WString;
```

Comment:
Is `WString` used?

Reply:
> its imported for the comment:

Thanks, however I'm not familiar, do the Javadocs require it? (I tried to look up this standard and could not find it).

Reply:
That makes sense.   I know projects like Apache won't accept PRs with unused imports.  They must use fully qualified Javadocs to uphold this rule.